exporter/metric/internal/statsd/conn_test.go

```diff
@@ -172,10 +138,10 @@ timer.B.D:%s|ms
 					}
 					const value = 123.456
 
-					checkpointSet.Add(cdesc, counterAgg(cdesc, value), labels...)
-					checkpointSet.Add(gdesc, gaugeAgg(gdesc, value), labels...)
-					checkpointSet.Add(mdesc, measureAgg(mdesc, value), labels...)
-					checkpointSet.Add(tdesc, measureAgg(tdesc, value), labels...)
+					checkpointSet.AddCounter(cdesc, value, labels...)
+					checkpointSet.AddGauge(gdesc, value, labels...)
+					checkpointSet.AddMeasure(mdesc, value, labels...)
+					checkpointSet.AddMeasure(tdesc, value, labels...)
```

Comment:
Nice refactoring.

Reply:
Thanks, the statsd exporter helped alot with this PR (: